mofchecker/__init__.py

```diff
@@ -656,9 +656,9 @@ def has_metal(self):
     def has_oms(self) -> bool:
         """True if the structure contains open metal sites (OMS).
```

Comment:
this starts immediately with "also" without explaining the heuristic for the default case